common/useFissionAuth.ts

```diff
@@ -0,0 +1,171 @@
+import { useEffect, useState } from 'react'
+
+import * as webnative from 'webnative'
+import * as webnativeFilecoin from 'webnative-filecoin';
+
+import * as C from '@common/constants';
+import * as R from '@common/requests';
+
+import Cookies from 'js-cookie';
+
+webnative.setup.debug({ enabled: true })
+
+export function useFissionAuth() {
+  const [state, setState] = useState<webnative.State>(null)
+  let fs;
+  let authScenario: webnative.Scenario | null = null;
+  let username: string = null;
+
+
+  /** Webnative Initialization
+   * Load webnative and configure permissions.
+   */
+
+  useEffect(() => {
+    async function getState() {
+      const result = await webnative.initialise({
+        permissions: {
+          // The Estuary token is stored in app storage
```

Comment:
Trying to avoid nits but

```
// NOTE(bgins)
// The Estuary token is stored in app storage
```

Is my preference so we always know who to forever pester about things ;-)

Reply:
Ha! I wrote those words, so I should be bothered to explain. :grin: 

I'll update myself as a commenter throughout.